Examples/php/cpointer/runme.php

```diff
@@ -1,40 +1,38 @@
 <?php
 
-	require "example.php";
-
 	# First create some objects using the pointer library.
 
 	print "Testing the pointer library\n";
 
-	$a = example::new_intp();
-	$b = example::new_intp();
-	$c = example::new_intp();
-	example::intp_assign($a,37);
-	example::intp_assign($b,42);
+	$a = new_intp();
+	$b = new_intp();
+	$c = new_intp();
+	intp_assign($a,37);
+	intp_assign($b,42);
 
 	print "	a = $a\n";
 	print "	b = $b\n";
 	print "	c = $c\n";
 
 	# Call the add() function wuth some pointers
-	example::add($a,$b,$c);
+	add($a,$b,$c);
 
 	# Now get the result
-	$r = example::intp_value($c);
+	$r = intp_value($c);
 
 	print "	37 + 42 = $r\n";
 
 	# Clean up the pointers
-	example::delete_intp($a);
-	example::delete_intp($b);
-	example::delete_intp($c);
+	delete_intp($a);
+	delete_intp($b);
+	delete_intp($c);
 
 	# Now try the typemap library
 	# This should be much easier. Now how it is no longer
 	# necessary to manufacture pointers.
 
 	print "Trying the typemap library\n";
-	$r = example::sub(37,42);
+	$r = sub(37,42);
```

Comment:
With the current wrappers `example::x` cannot be called with `x` . Since the functions were all non-class functions. I've wrapped such that, they can be accessed as `x` and that there's no need for class example. But I didn't get what you mentioned about `example` class for compatibility. And I'll do look at namespaces. I've got a reference here of namespaces, [HERE](https://stackoverflow.com/questions/24687502/can-i-put-php-extension-classes-functions-etc-in-a-namespace)